exercises/Ex4RandomNumber.js

```diff
@@ -0,0 +1,21 @@
+function yourLucky(yourInput){
+    var randomInt = Math.floor(Math.random() * 10);
+    var userNumber = parseInt(prompt(yourInput + "Please enter a number from 1 to 10.", ""));
+    if (userNumber > 0 && userNumber <= 10){
+        if(randomInt == userNumber){
+            alert("Good Work.");
+        }
+        else{
+            alert("Not matched.");
+        }
+    }
+    else if (isNaN(userNumber)){
+        yourInput = "Your input is not a number. "
+        return yourLucky(yourInput);
+    }
+    else {
+        yourInput = "Your input is incorrect. "
```

Comment:
Not happy as a User with such message. 
"Please enter an integer number between 1 and 10".